diff --git a/Modules/Video/BridgeOpenCV/include/itkOpenCVVideoIO.h b/Modules/Video/BridgeOpenCV/include/itkOpenCVVideoIO.h
index 1570a182..96a70f17 100644
--- a/Modules/Video/BridgeOpenCV/include/itkOpenCVVideoIO.h
+++ b/Modules/Video/BridgeOpenCV/include/itkOpenCVVideoIO.h
@@ -168,7 +168,7 @@ public:
                       const std::vector<SizeValueType> & dim,
                       const char *                       fourCC,
                       unsigned int                       nChannels,
-                      IOComponentType                    componentType);
+                      IOComponentEnum                    componentType);
 
 protected:
   OpenCVVideoIO();
diff --git a/Modules/Video/BridgeOpenCV/src/itkOpenCVVideoIO.cxx b/Modules/Video/BridgeOpenCV/src/itkOpenCVVideoIO.cxx
index 7d7db3b8..8ce71c8f 100644
--- a/Modules/Video/BridgeOpenCV/src/itkOpenCVVideoIO.cxx
+++ b/Modules/Video/BridgeOpenCV/src/itkOpenCVVideoIO.cxx
@@ -436,7 +436,7 @@ OpenCVVideoIO::SetWriterParameters(TemporalRatioType                  fps,
                                    const std::vector<SizeValueType> & dim,
                                    const char *                       fourCC,
                                    unsigned int                       nChannels,
-                                   IOComponentType                    componentType)
+                                   IOComponentEnum                    componentType)
 {
   if (this->m_ReaderOpen || this->m_WriterOpen)
   {
@@ -444,7 +444,7 @@ OpenCVVideoIO::SetWriterParameters(TemporalRatioType                  fps,
   }
 
   // Make sure componentType is acceptable (right now we only support char)
-  if (componentType != UCHAR)
+  if (componentType != IOComponentEnum::UCHAR)
   {
     itkExceptionMacro("OpenCV IO only supports writing video with pixels of UCHAR");
   }
diff --git a/Modules/Remote/GrowCut.remote.cmake b/Modules/Remote/GrowCut.remote.cmake
new file mode 100644
index 0000000000..083f3c52a4
--- /dev/null
+++ b/Modules/Remote/GrowCut.remote.cmake
@@ -0,0 +1,7 @@
+itk_fetch_module(GrowCut
+"ITKGrowCut segments a 3D image from user-provided foreground and background seeds."
+  MODULE_COMPLIANCE_LEVEL 3
+  GIT_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITKGrowCut.git
+  GIT_TAG 7e9959f46bf40fe23d9ae30364055ae29e6059d0
+  )
+
